ddl/job_table.go

```diff
@@ -445,7 +445,11 @@ func (d *ddl) MoveJobFromQueue2Table(force bool) error {
 		}
 		t := meta.NewMeta(txn)
 		isConcurrentDDL, err := t.IsConcurrentDDL()
-		if !force && (isConcurrentDDL || err != nil) {
+		if !inBootstrap && (isConcurrentDDL || err != nil) {
```

Comment:
even inBootstrap, we still need to check the error.

Reply:
the error can be ignored because `isConcurrentDDL` is only used here and if in `inBootstrap` we shall always think `isConcurrentDDL` is false, So that the error does not matter